### DISCUSSION TITLE
Cannot login using gitlab

### DISCUSSION THREAD

**Discussion by shvamabps** [vercel/community#1837] [Help] [answered]
![image](https://user-images.githubusercontent.com/37547983/224535097-2f4f28f1-d02a-4052-bbb0-390cac284382.png)
this is the error shown in console when trying to login using gitlab

![image](https://user-images.githubusercontent.com/37547983/224535131-154ed1a6-941b-40f6-9f6f-d4585f3dabc4.png)

![image](https://user-images.githubusercontent.com/37547983/224535141-021d57ac-9837-4c9b-adc6-198e08b92d50.png)

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include your GitLab username. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
